### ISSUE TITLE
Custom customer tab exception

### ISSUE THREAD

**Issue by nl-it** [bitnami/bitnami-docker-magento#154]
**Description**

I'm trying to create a simple customer tab to display information and I'm getting the following exception:

Exception #0 (Magento\Framework\Exception\ValidatorException): Path "/bitnami/magento/htdocs/app/code/CSW/CustomerTab/view/frontend/templates/customer/account/customtab.phtml" cannot be used with directory "/opt/bitnami/magento/htdocs/"

**Steps to reproduce the issue:**

1. Run a instance of bitnami magento docker image
2. Install the Magento Sample Data 
3. Install the attached module into (/bitnami/magento/htdocs/app/code/CSW)
4. Run magento setup:upgrade
5. Run magento setup:di:compile
6. Run magento cache:flush
7. Login as a customer (you can use the roni_cost@example.com sample account)
8. Hit the "Super Awesome" option tab on the left

**Describe the results you received:**

An exception is thrown.

**Describe the results you expected:**

The template should be rendered.

**Additional information you deem important (e.g. issue happens only occasionally):**

If I use another template e.g.

the page is rendered.

**Version**

- Output of `docker version`:

```
 Version:           19.03.8
 API version:       1.40
 Go version:        go1.12.17
 Git commit:        afacb8b
 Built:             Wed Mar 11 01:23:10 2020
 OS/Arch:           windows/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.8
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.17
  Git commit:       afacb8b
  Built:            Wed Mar 11 01:29:16 2020
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          v1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

- Output of `docker info`:

```
Client:
 Debug Mode: false

Server:
 Containers: 4
  Running: 4
  Paused: 0
  Stopped: 0
 Images: 18
 Server Version: 19.03.8
 Storage Driver: overlay2
  Backing Filesystem: <unknown>
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 4.19.76-linuxkit
 Operating System: Docker Desktop
 OSType: linux
 Architecture: x86_64
 CPUs: 3
 Total Memory: 3.848GiB
 Name: docker-desktop
 ID: NHAZ:DTQZ:YHHU:45WQ:244E:HCRQ:EXZM:PX6J:ZZYD:25GJ:GVD3:GI2O
 Docker Root Dir: /var/lib/docker
 Debug Mode: true
  File Descriptors: 76
  Goroutines: 98
  System Time: 2020-07-16T13:09:31.820306077Z
  EventsListeners: 4
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: true
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
 Product License: Community Engine

```

- Output of `docker-compose version
` (if applicable):

```
docker-compose version 1.25.5, build 8a1c60f6
[CustomerTab.zip](https://github.com/bitnami/bitnami-docker-magento/files/4931657/CustomerTab.zip)
docker-py version: 4.1.0
CPython version: 3.7.4
OpenSSL version: OpenSSL 1.1.1c  28 May 2019
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
[CustomerTab.zip](https://github.com/bitnami/bitnami-docker-magento/files/4931709/CustomerTab.zip)

**Comment by nl-it** [comment]
The same also occurs following the Magento Tutorial => https://devdocs.magento.com/guides/v2.4/ext-best-practices/extension-coding/example-module-adminpage.html

**Comment by rafariossaa** [comment]
Hi, 
  This issue look similar to this other one: https://github.com/bitnami/bitnami-docker-magento/issues/113 , 
  Could you take a look at it ?

**Comment by nl-it** [comment]
The issue was occuring on the **docker.io/bitnami/magento:2-debian-10** image. I've changed the docker-compose yaml file to use the **latest** and it is fixed.

**Comment by rafariossaa** [comment]
Hi,
  I this was fixed in container version 2.3.5, this is the note about the change:
 https://github.com/bitnami/bitnami-docker-magento/#user-content-235-debian-10-r57
